integration-tests/integration/features/cases-in-progress-list-filter.feature

```diff
@@ -108,19 +63,17 @@ Feature: Cases In progress List filters
       | English Madden | Adjourned          | Previously known | Attempt theft from the person of another | 5 Sep 2023 | Move to resulted |
 
     When I click the "Outcome type" filter button
-    And I select the "OTHER" filter
+    And I select the "Other" filter
     And I click the "Outcome type" filter button
     And I click the "Apply filters" button
 
     And I should see the "outcomeType" query have the value "OTHER"
 
-    Then I should see a tab with text "In progress (0)"
-
-    And I should see a count of "0 cases"
+#    Then I should see a tab with text "In progress (0)"
```

Comment:
Is this going to be put back when we do the dynamic counts?

Reply:
Yes

Reply:
And I should see a count of "0 cases" - will not be back as this is not as part of the design/AC